src/Spryker/Zed/UserMerchantPortalGui/Communication/Controller/MyAccountController.php

```diff
@@ -23,14 +23,21 @@ class MyAccountController extends AbstractController
     /**
      * @var string
      */
-    protected const URL_MERCHANT_MY_ACCOUNT = '/user-merchant-portal-gui/my-account';
+    protected const ROUTE_MERCHANT_MY_ACCOUNT = '/user-merchant-portal-gui/my-account';
```

Comment:
I believe you don't need these changes for backport
otherwise it means 1.12 has new constants and 2.0.0 has old ones
we don't need to backport refactoring